src/main/java/eu/dissco/backend/service/RabbitMqPublisherService.java

```diff
@@ -0,0 +1,27 @@
+package eu.dissco.backend.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class RabbitMqPublisherService {
+
+  @Value("${rabbitmq.mas-exchange-name:mas-exchange}")
+  private String exchangeName = "mas-exchange";
+
+  private final ObjectMapper mapper;
+  private final RabbitTemplate rabbitTemplate;
+
+  public void sendObjectToQueue(String routingKey, Object object) throws JsonProcessingException {
+    log.info("Sending to exchange with routing key: {} and with object: {}", routingKey, object);
```

Comment:
if you're sending the whole object, maybe make it a debug?

Reply:
:+1: